apps/host/src/pages/notice-create/notice-form.tsx

```
@@ -0,0 +1,166 @@
+              currentCount={images.length}
+              maxCount={MAX_IMAGES}
+              onFilesChange={handleImagesAdd}
+            />
+            <div
+              ref={scrollRef}
+              className={styles.imageList}
+              onMouseDown={onDragStart}
+              onMouseMove={onDragMove}
+              onMouseUp={onDragEnd}
+              onMouseLeave={onDragEnd}
+              role='list'
```

Biome (2.4.6)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

(lint/a11y/useSemanticElements)